public/app/features/alerting/unified/NotificationPolicies.test.tsx

```diff
@@ -29,6 +29,8 @@ import { getAllDataSources } from './utils/config';
 import { ALERTMANAGER_NAME_QUERY_KEY } from './utils/constants';
 import { DataSourceType, GRAFANA_RULES_SOURCE_NAME } from './utils/datasource';
 
+import 'core-js/stable/structured-clone';
```

Comment:
It's mainly a matter of preference for me to use built-in functions instead of external ones. Unfortunately, we use a quite old Node version, so a polyfill is needed in tests